ouroboros-network/test/Test/Ouroboros/Network/Testnet.hs

```diff
@@ -2104,6 +2107,12 @@ prop_diffusion_cm_valid_transition_order defaultBearerInfo diffScript =
          . groupConns id abstractStateIsFinalTransition
          $ abstractTransitionEvents
 
+prop_unit_4258 :: Property
```

Comment:
nice :), you could include a link to the issue (or just explain that it's issue #4258).

Reply:
I will :)